File: Java/Ejercicios01/Persona.java
public abstract class Persona {
	private String nombre;
	private Fecha fecNac;
	private boolean sexo;
	
	public Persona(String nombre, Fecha fecNac, boolean sexo) {
		this.nombre = nombre;
		this.fecNac = fecNac;
		this.sexo = sexo;
	}
	public Persona() {}
	public String getNombre() {
		return nombre;
	}
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	public Fecha getFecNac() {
		return fecNac;
	}
	public void setFecNac(Fecha fecNac) {
		this.fecNac = fecNac;
	}
	public boolean getSexo() {
		return sexo;
	}
	public void setSexo(boolean sexo) {
		this.sexo = sexo;
	}
	
	public abstract boolean esDiabetico();
	
	public abstract boolean esHipertenso();
	
}

File: PooTarea03/src/main/kotlin/Persona.kt
class Persona : CompaniaTelf, Mensajes {
    private var nombre: String = ""
    private var fecNac: Fecha? = null
//    se utiliza `?` para indicar que fecNac puede ser nulo
//    (ya que en Kotlin, por defecto, las propiedades no pueden tener valores nulos).

    constructor(compania: String, numero: String, nombre: String, fecNac: Fecha) : super(compania, numero) {
        this.nombre = nombre
        this.fecNac = fecNac
    }

    constructor()

    fun setNombre(nombre: String) {
        this.nombre = nombre
    }

    fun getNombre(): String {
        return nombre
    }

    fun setFecNac(fecNac: Fecha?) {
        this.fecNac = fecNac
    }

    fun getFecNac(): Fecha? {
        return fecNac
    }

    override fun enviarMensaje() {
        println("  $nombre ha enviado un mensaje")
    }

    override fun recibirMensaje() {
        println("  $nombre ha recibido un mensaje")
    }
}